addons/mail/static/src/models/notification_group/notification_group.js

```diff
@@ -67,6 +67,14 @@ function factory(dependencies) {
             return `${this.modelName}_${data.id}`;
         }
 
+        _computeDate() {
+            return moment.max(this.notifications.map((notification) => notification.message.date));
+        }
+
+        _computeSequence() {
+            return -Math.max(...this.notifications.map((notification) => notification.id));
```

Comment:
It would probably be more clear with using `min` and no negation.

Reply:
At this point, I m not even sure this could be called a "sequence" anymore. It's more a "weight" to me since the number do not directly follow each other.

@alexkuhn Is this change compatible with your definition of the "sequence" ?